packages/test-renderer/src/__tests__/setupTests.ts

```diff
@@ -0,0 +1,5 @@
+import { extend } from '@react-three/fiber'
+import * as THREE from 'three'
+
+// Extend catalogue for render API in tests.
+extend(THREE)
```

Comment:
This should be done automatically for the test renderer since `render` is called internally. Seems to be an unneeded breaking change for tests.

This PR addresses some regressions with the `act` API, but this might follow.